packages/block-library/src/separator/style.scss

```diff
@@ -17,7 +18,7 @@
 
 		&::before {
 			content: "\00b7 \00b7 \00b7";
-			color: $dark-gray-900;
+			color: currentColor;
```

Comment:
I know we moved away from `currentColor` in other places, probably because it's deprecated?